tests/spark_autologging/test_pyspark_ml_autologging_custom_allowlist.py

```diff
@@ -0,0 +1,24 @@
+import mlflow
+import pytest
+from pkg_resources import resource_filename
+
+from tests.spark_autologging.utils import spark_session  # pylint: disable=unused-import
+
+
+@pytest.mark.parametrize(
+    "spark_session",
+    [
+        {
+            "spark.mlflow.pysparkml.autolog.logModelAllowlistFile": resource_filename(
+                __name__, "custom_log_model_allowlist.txt"
+            )
+        }
+    ],
+    indirect=["spark_session"],
+)
+def test_custom_log_model_allowlist(spark_session):  # pylint: disable=unused-argument
```

Comment:
I put this in a separate test file because `spark_session` is module scope fixture, and once `spark_session` created, the `spark.mlflow.pysparkml.autolog.logModelAllowlistFile` config cannot be changed in runtime.

keep `spark_session` being module scope will help speedup test because initializing a new context took time.